solidity/.tsconfig-eslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [".eslintrc.js", "**/*.js", "**/*.ts", "**/*.tsx"],
+  "exclude": ["node_modules"]
+}
```

Comment:
Shouldn't we symlink to `./node_modules/@thesis-co/eslint-config/.tsconfig-eslint.json` just like we do [here](https://github.com/keep-network/keep-core/blob/main/solidity/random-beacon/.tsconfig-eslint.json)?

Reply:
Done.